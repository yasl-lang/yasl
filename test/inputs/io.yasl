const f = io.tmpfile()

f->write("12345")

f->seek()

echo f->read('a')

f->seek(.end)

f->write('\nabcde')

f->seek()

echo f->read('l')
echo f->read('l')
