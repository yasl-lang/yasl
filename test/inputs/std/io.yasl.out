12345
12345
abcde
